src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -756,6 +778,21 @@ type DepartureRow implements Node & PlaceInterface {
     ): [Stoptime]
 }
 
+"""
+Direction on a route. In practice,
+this means the trips/patterns of a route that have the defined direction.
+"""
+type DirectionOnRoute {
```

Comment:
Maybe it would be useful to expose the affected trips/patterns through this type.

Reply:
Removed this type and instead generated Pattern selectors from the DirectionAndRoute entity selector.